app/src/main/java/com/example/tasky/feature_agenda/domain/model/AgendaResponse.kt

```diff
@@ -1,17 +1,11 @@
 package com.example.tasky.feature_agenda.domain.model
 
-import com.example.tasky.agenda_details.domain.model.EventResponse
-import com.example.tasky.agenda_details.domain.model.ReminderResponse
-import com.example.tasky.agenda_details.domain.model.TaskResponse
+import com.example.tasky.agenda_details.domain.model.Event
+import com.example.tasky.agenda_details.domain.model.Reminder
+import com.example.tasky.agenda_details.domain.model.Task
 
 data class AgendaResponse(
-    val events: List<EventResponse> = emptyList(),
-    val tasks: List<TaskResponse> = emptyList(),
-    val reminders: List<ReminderResponse> = emptyList()
-)
-
-data class SyncAgendaResponse(
-    val deletedEventIds: List<String> = emptyList(),
-    val deletedTasksIds: List<String> = emptyList(),
-    val deletedReminderIds: List<String> = emptyList()
+    val events: List<Event> = emptyList(),
+    val tasks: List<Task> = emptyList(),
+    val reminders: List<Reminder> = emptyList()
```

Comment:
This class is not needed in domain anymore. You just need it in data to receive and parse the response from the API, but then the rest of the app can just work with a `List<AgendaItem>`